blueocean-pipeline-api-impl/pom.xml

```diff
@@ -67,7 +67,7 @@
         <dependency>
             <groupId>org.jenkins-ci.plugins.workflow</groupId>
             <artifactId>workflow-durable-task-step</artifactId>
-            <version>2.0</version>
+            <version>2.4</version>
```

Comment:
I had to bump it to be able to start all plugins correctly. (This makes the docker image a great test to ensure all plugins dependencies are coherent)